File: CTDK_Python/087/_087.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = x
t = x
dau = -1
i = 3
while(i <= 2 * n + 1):
    t = t * x * x
    s = s + dau * t
    i = i + 2
    dau = -dau

print("Tong la: ", s)

File: CTDK/087/087.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;

	float s = 0;
	float t = x;
	int dau = 1;
	int i = 0;
	while (i <= n)
	{
		s = s + dau * t;
		t = t * x * x;
		dau = -dau;
		i++;
	}
	cout << "Tong la: " << s << endl;
	return 0;
}